addon/ember-internals.js

```diff
@@ -185,11 +185,14 @@ function sameRouteState(a, b) {
 // This lets us invoke an outlet with an explicitly passed outlet
 // state, rather than inheriting it implicitly from its context.
 export var StaticOutlet = Ember.OutletView.superclass.extend({
-  tagName: '',
+  //tagName: '',
```

Comment:
Why is this commented out? This would be a breaking change for people's apps due to layout changing.